Makefile

```diff
@@ -160,9 +160,9 @@ clean-cache:
 
 bundle: ## Installs dependencies using bundler. Run this after you make some changes to Gemfile.
 bundle: Gemfile.prod Gemfile
```

Comment:
I did not know about this thanks!
I think it is available only on 1.11+ 
rhscl install 1.10 by default. We spoke about that with @guicassolato and agree we should use at least 1.111
So 👍